Cargo.toml

```diff
@@ -13,16 +13,16 @@ edition = "2018"
 [badges]
 travis-ci = { repository = "mullvad/pfctl-rs" }
 
-
 [dependencies]
-errno = "0.2"
-error-chain = "0.12"
-ioctl-sys = "0.6.0"
+derive_builder = "0.20"
+errno = "0.3"
+ioctl-sys = "0.8"
+ipnetwork = "0.20"
 libc = "0.2.29"
-derive_builder = "0.9"
-ipnetwork = "0.16"
+thiserror = "1"
 
 [dev-dependencies]
 assert_matches = "1.1.0"
-uuid = { version = "0.8", features = ["v4"] }
+error-chain = "0.12"
 scopeguard = "1.0"
+uuid = { version = "1", features = ["v4"] }
```

Comment:
Please separate the dependency upgrades not related to swapping out `error-chain` to a separate PR. I prefer if PRs are as focused as possible on one thing.

Reply:
I'll have to wait for the merge then. That's actually very annoying if you accept only one thing per PR.

Reply:
If we have issues/complications it's going to be significantly easier to find what the cause is if changes are separated into smaller logical chunks. Also easier to revert single things instead of a blob of unrelated things. Also way easier to review and reason about.

Reply:
If you submit small easy to review PRs I can review and merge them fast.